Makefile

```diff
@@ -53,11 +53,15 @@ build:
 	  `pwd | sed -e "s|${GOPATH_ROOT}/src/||"`/$$i; \
 	done
 
-build/mackerel-check:
+build/mackerel-check: $(patsubst %,depends_on,$(GOOS)$(GOARCH))
```

Comment:
It looks magical for me.  Can you comment this mean?  Maybe I think this code mean that force to rebuild build/mackerel-check if GOOS or GOARCH is specified.

Reply:
I added.